openupgradelib/openupgrade.py

```diff
@@ -2372,6 +2372,12 @@ def format_message(f):
         globaldict = {'uid': env.uid}
         if version_info[0] < 14:
             globaldict.update({'time': time})
+        elif version_info[0] > 14:
```

Comment:
```suggestion
        if version_info[0] >= 13:
```
https://github.com/odoo/odoo/blob/13.0/odoo/addons/base/models/ir_filters.py#L41

Reply:
Okey, done

Reply:
Remove the else part also, as it should apply on v13 as well

Reply:
Sorry, I edited